### PR TITLE
DE-718-find-more-information-on-customfields

### PR DESCRIPTION
**Pull request by SBurwash**
### Ticket ([DE-718](https://potloc.atlassian.net/browse/DE-718))

> In the front end of Wrike we know who modified a customfield and on which date, can you check if it is possible to implement in the database? 

---
_from `tiguidou` with :heart:_

### CODE REVIEW

**Review by charles2cb — APPROVED** [review]

File: tap_wrike/streams.py [hunk 1]
```diff
@@ -104,6 +104,14 @@ class dotdict(dict):
         "replication_method": "INCREMENTAL",
         "replication_key": "change_datetime",
     }),
+    "data_export_api_work_custom_field_history": dotdict({
+        "csv": True,
+        "path": "work_custom_field_history",
+        "params": {},
+        "key_properties": ["work_id", "user_resource_id", "old_custom_field_value", "custom_field_id", "change_datetime"],
```
Comment (jeromepl) [inline tap_wrike/streams.py]: Are we sure this is the right unique index for this table? It seems a bit strange 🤔

**Review by jeromepl — APPROVED** [review]

File: tap_wrike/streams.py (hunk 1 above)
Comment (SBurwash) [reply tap_wrike/streams.py]: Good catch; we need to remove the 2 value columns

Comment (SBurwash) [reply tap_wrike/streams.py]: @jeromepl what does the key_properties entry actually mean? Primary keys, or propoerties to be outputed?

Comment (jeromepl) [reply tap_wrike/streams.py]: The primary key. In most cases it's just a single ID column, but sometimes its a more complex composite like it seems to be here

Comment (SBurwash) [reply tap_wrike/streams.py]: Thanks! I was able to find the right values from [here](https://developers.wrike.com/schema-reference/)

**Pull request merged**